src/applications/disability-benefits/526EZ/pages/primaryAddress.js

```diff
@@ -578,10 +587,8 @@ export const primaryAddressSchema = {
       properties: {
         mailingAddress: addressSchema,
         primaryPhone: {
-          type: 'string'
-        },
-        secondaryPhone: {
-          type: 'string'
+          type: 'string',
+          pattern: '^\\d{10}$'
```

Comment:
Will this permit dashes?  Should we allow dashes on the front end and remove them in a backend transformation?

Reply:
So actually the phone number widget uses local state and sanitizes it before sending the number to redux. This means that we get to use a simple pattern here, while still allowing parens, spaces, and dashes in the front end.